src/utils/veramoUtils.ts

```diff
@@ -65,23 +71,27 @@ export async function veramoCreateVP(
   const identifier = await veramoImportMetaMaskAccount();
   //Get Veramo agent
   const agent = await getAgent();
-  //Get VC from state
-  const vc = await agent.getVC({ id: vc_id });
+  let vc;
+  try {
+    vc = await agent.getVC({ store: 'snap', id: vc_id });
```

Comment:
Does `agent.getVC` throw an error if the VC is not found or does it return `null` ? 
Maybe we don't need the try and catch -> or it would be probably better to move the try and catch into the VC manager and in the case of an error throw `null` ?

Reply:
agent.VC throws an error if it fails to retrieve a VC